### PR TITLE
Added Course page

### PR DESCRIPTION
**Pull request by MGafitescu**
Created the Course html page and added the necessary classes in the
style.css files. The course page is the template page for all the tasks
in a course.

### CODE REVIEW

File: course.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	<meta name="author" content="Seniuc Bogdan, Popescu Georgian Stefan, Gafitescu Petru Marian, Pomohaci Alexandru">
+</head>
+
+<body>
+	<nav>
+		<div class="container">
+			<a href="#" class="logo">SkIns</a>
+
+			<a href="#" class="active">Learn</a>
+			<a href="#">Tasks</a>
```
Comment (bogsen) [inline course.html]: Make this active instead of the other page

File: style.css [hunk 2]
```diff
@@ -57,55 +57,67 @@ body > .container {
...
-	max-width: 90%;
+	max-width: calc(100% - 4rem);
 }
 
 .box {
 	background-color: white;
 	box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.25);
 	border-radius: 6px;
-	height: 10rem;
+  padding: 1rem;
```
Comment (bogsen) [inline style.css]: Indentation not consistent (entire file)

**Review by bogsen — CHANGES_REQUESTED** [review]
👍🏻

File: style.css [hunk 3]
```diff
@@ -57,55 +57,67 @@ body > .container {
...
+	display: table;
+	clear: both;
 }
 
 #description-image {
 	float: right;
 	object-fit: cover;
 	max-width: 12rem;
-	max-height: 100%;
+	max-height: 100%;;
```
Comment (GeoStefan) [inline style.css]: There are too many ";".

File: style.css [hunk 4]
```diff
@@ -188,20 +206,189 @@ h3 {
 	right: 0;
 }
 
-@media screen and (min-width: 70rem) {
-	.description {
-		max-width: 70rem;
+/*Display the course section boxes more larger*/
+/*
+@media screen and (min-width: 40rem) {
+		.flex-container {
```
Comment (bogsen) [inline style.css]: Indentation

**Review by bogsen — APPROVED** [review]

**Pull request merged**